dracut/30ignition/ignition-setup-user.sh

```diff
@@ -14,11 +14,17 @@ copy_file_if_exists() {
 destination=/usr/lib/ignition
 mkdir -p $destination
 
-# We will support a user embedded config in the boot partition
-# under $bootmnt/ignition/config.ign. Note that we mount /boot
-# but we don't unmount boot because we are run in a systemd unit
-# with MountFlags=slave so it is unmounted for us.
-bootmnt=/mnt/boot_partition
-mkdir -p $bootmnt
-mount /dev/disk/by-label/boot $bootmnt
-copy_file_if_exists "${bootmnt}/ignition/config.ign" "${destination}/user.ign"
+if command -v is-live-image >/dev/null && is-live-image; then
+    # Live image. If the user has supplied a config.ign via an appended
+    # initrd, put it in the right place.
+    copy_file_if_exists "/config.ign" "${destination}/user.ign"
```

Comment:
this is fine too, but why not have it already in the correct place in the second initrd? Simplicity of creating the initrd?

Reply:
Asking users to hand-create `usr/lib/ignition/user.ign` is awkward, and there are other things in that directory that we don't want them to create.  Putting the config at the root saves a needless directory tree during cpio creation.

Reply:
SGTM, mostly was just curious.